CONTRIBUTING.md

```diff
@@ -28,18 +29,18 @@ If you intend to add new features or suggest major changes to this module, check
 
 If you’re only fixing a bug, it’s fine to submit a pull request right away but we still recommend to file an issue detailing what you’re fixing. This is helpful in case we don’t accept that specific fix but want to keep track of the issue.
 
-## Contribution Prerequisites
+## Contribution Requirements
 
-- You have the latest version of [Node](https://nodejs.org) and [Git](https://git-scm.com) installed
+- You have the latest version of [Node](https://nodejs.org), [Git](https://git-scm.com), and installed.
 - You will be working on one item at a time.
 - If you do not have it yet, fork the repository. Clone it if you will work locally.
 - If you have already forked and clone the repository, make sure that it is in sync with the upstream repository ([Syncing a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork)).
-- Local backend server already setup and running. (This API backend is needed before running this project. The instructions on how to setup this API locally can be found [here](https://github.com/sws2apps/sws2apps-api/blob/main/CONTRIBUTING.md).)
+- Local [API backend]([url](https://github.com/sws2apps/sws2apps-api)) server already setup and running. The backend is used mainly for login and sync functionality. See our [backend local setup guide](https://github.com/sws2apps/sws2apps-api/blob/main/CONTRIBUTING.md).)
```

Comment:
@ux-git: Although this is working, could we please change it into: `[API backend](https://github.com/sws2apps/sws2apps-api)`? Thanks